tests/integration/components/pikaday-input-test.js

```diff
@@ -23,6 +23,20 @@ const getDisabledDayCB = weekendDay => e => {
 module('Integration | Component | pikaday-input', function(hooks) {
   setupRenderingTest(hooks);
 
+  hooks.beforeEach(function() {
```

Comment:
Also calling it out here; please rebase this PR rather than merging `master` in. These test changes are not something I want in the PR.